src/styles/text.tsx

```diff
@@ -42,6 +42,14 @@ export const H4 = styled.h4<{ $color?: string }>`
   margin: 0;
 `;
 
+export const H5 = styled.h5<{ $color?: string }>`
```

Comment:
For our headers, we try to have `H1` have the largest font size, and `H2` would have the next largest font size, and so on. Using this convention,  `H5` would need to have a smaller font size, so we should probably rename this. Additionally, since it seems like this large font size is only being used in the home page so far, we can move this to the `styles.tsx` file corresponding to the home page.